frigate/detectors/plugins/openvino.py

```diff
@@ -17,50 +17,119 @@ class OvDetectorConfig(BaseDetectorConfig):
     type: Literal[DETECTOR_KEY]
     device: str = Field(default=None, title="Device Type")
 
-
 class OvDetector(DetectionApi):
     type_key = DETECTOR_KEY
 
     def __init__(self, detector_config: OvDetectorConfig):
         self.ov_core = ov.Core()
         self.ov_model = self.ov_core.read_model(detector_config.model.path)
+        self.ov_model_type = detector_config.model.model_type
+
+        self.h = detector_config.model.height
+        self.w = detector_config.model.width
 
         self.interpreter = self.ov_core.compile_model(
             model=self.ov_model, device_name=detector_config.device
         )
+
         logger.info(f"Model Input Shape: {self.interpreter.input(0).shape}")
         self.output_indexes = 0
+
         while True:
             try:
                 tensor_shape = self.interpreter.output(self.output_indexes).shape
                 logger.info(f"Model Output-{self.output_indexes} Shape: {tensor_shape}")
+                logger.info(f"Model Output-{self.output_indexes} Shape: {tensor_shape}")
```

Comment:
Seems like this line was duplicated?

Reply:
Haha yep, it sure is a duplicate. I'll fix that.